terraform/bind/main.tf

```diff
@@ -1,6 +1,10 @@
 locals {
   instance_id = "ses-${substr(sha256(var.instance_name), 0, 16)}"
   user_name = "${local.instance_id}-${var.user_name}"
+  subscribe_bounce_notification = (var.bounce_topic_arn != "" && var.notification_webhook != "")
+  subscribe_complaint_notification = (var.complaint_topic_arn != "" && var.notification_webhook != "")
+  subscribe_delivery_notification = (var.delivery_topic_arn != "" && var.notification_webhook != "")
+  subscribed_webhook = ((local.subscribe_bounce_notification || local.subscribe_complaint_notification || local.subscribe_delivery_notification) ? var.notification_webhook : null)
```

Comment:
I'm a little confused about the intent here... Are users expected to pick EITHER topic notifications OR webhook notifications? And if so, is that a technical constraint? Just wondering if there's a reason to restrict the choice that way.

Reply:
Docs would help answer this a bit better, so I'll also improve those.

You actually use both. SES sends the feedback notifications to SNS topics. That happens in the provision step.

To actually receive those notifications out of SNS, the bind step is registering a HTTPS webhook onto each of those SNS topics. The bind code says "if I have both a topic ARN, and a webhook, then I'll connect the two through a subscription." The code allows separate ARNs for the SNS topics, and allows any number of them to be set, but right now funnels all of them to a single webhook. That's how notify is currently setup.

The output param here is kinda just a flag back to the app that lets the service say "yes, I successfully registered this callback to receive the notifications out of SNS"